test-framework/sudo-compliance-tests/src/sudoers.rs

```diff
@@ -18,7 +18,10 @@ fn cannot_sudo_with_empty_sudoers_file() -> Result<()> {
 
 #[test]
 fn cannot_sudo_if_sudoers_file_is_world_writable() -> Result<()> {
-    let env = EnvBuilder::default().sudoers_chmod("446").build()?;
+    let env = EnvBuilder::default()
```

Comment:
>  in the original test sudo-rs would fail because the sudoers file it read contained no rules.

I think that's indicative that sudoers is doing the checks in different order: it's checking the permissions first and then parsing the sudoers file.

Reply:
In the original code it was not checking the permissions at all; the test case tested for `sudo true`, which is not permitted by an empty sudoers file.

Reply:
> that sudoers is doing the checks in different order

sorry, typo. what I meant to say is that original sudo is checking permissions first and then parsing the file. as original sudo sees that the perms are wrong it exits before even parsing the (empty) file. if you need to modify the test case to make sudo-rs (as modified by this PR) pass the test then, to me, that indicates sudo-rs is parsing the file and then checking the permissions. in which case sudo-rs behavior is different from the original sudo.

Reply:
> you can see it had no #[ignore] tag

I missed that. then yeah, the change makes sense